src/main/java/com/github/manolo8/darkbot/core/manager/PetManager.java

```diff
@@ -283,6 +285,53 @@ private void updateGear(Gear module, Gear subModule) {
         currentSubModule = module == null ? null : subModule;
     }
 
+    private void updatePetStats(long elementsListAddress) {
+        if (pet.removed) return;
+        for (PetStatsType type : PetStatsType.values())
+            petStats.computeIfAbsent(type, PetStats::new).update(elementsListAddress, type.id);
+    }
+
+    public class PetStats {
+        private double curr, total;
+
+        public double getCurr() { return curr; }
+        public double getTotal() { return total; }
```

Comment:
Avoid one-liner getter/setters, a good IDE will automatically display it as a one-line for you

Reply:
done